### DISCUSSION TITLE
more guides in the main website

### DISCUSSION THREAD

**Discussion by aliibrahim123** [GrapesJS/grapesjs#3985] [Ideas]
hallo @artf,
can you make more guides in the main website,
I noticed that there are so many frequently asked questions that can be put in it

**Comment by artf** [comment]
Yeah, a FAQ page is definitely something to add, but what exactly are you referring to?

**Reply by aliibrahim123** [reply]
one of the most asked questions is how to make a block filter

**Reply by aliibrahim123** [reply]
thinks like this would be amazing
